src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -151,4 +151,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Info,
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor DoNotUseIHeaderDictionaryAdd = new(
+        "ASP0019",
+        new LocalizableResourceString(nameof(Resources.Analyzer_HeaderDictionaryAdd_Title), Resources.ResourceManager, typeof(Resources)),
+        new LocalizableResourceString(nameof(Resources.Analyzer_HeaderDictionaryAdd_Message), Resources.ResourceManager, typeof(Resources)),
+        "Usage",
+        DiagnosticSeverity.Warning,
+        isEnabledByDefault: true,
+        helpLinkUri: "https://aka.ms/aspnet/analyzers");
```

Comment:
Is there an issue tracking documentation for analyzers?

I see ASP0015, ASP0016, ASP0017, and ASP0018 with the same help link are not yet documented.

Reply:
@david-acker You can file a docs issue in the docs repo here https://github.com/dotnet/AspNetCore.Docs/issues.

If you're so inclined, you can also submit the doc for this by updating [this page](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/diagnostics/code-analysis.md).